README.md

```diff
@@ -1,21 +1,2 @@
 # BnB Explorer Docker
-Local Docker development environment for BnB Explorer
```

Comment:
Same thing here, I think we should keep the run instructions as part of the readme. Its always useful to include those in the readme, without having to consult a wiki

Reply:
@cdeiac Done